Firebase/Auth/Source/FIRUser.m

```diff
@@ -845,9 +845,18 @@ - (void)getIDTokenResultForcingRefresh:(BOOL)forceRefresh
 - (FIRAuthTokenResult *)parseIDToken:(NSString *)token error:(NSError **)error {
   *error = nil;
   NSArray *tokenStringArray = [token componentsSeparatedByString:@"."];
+
   // The token payload is always the second index of the array.
   NSMutableString *tokenPayload = [[NSMutableString alloc] initWithString:tokenStringArray[1]];
```

Comment:
Consider this as more compact and readable:

NSString inputToken = tokenStringArray[1];
NSMutableString *tokenPayload = [inputToken stringByReplacing....